src/MLOps.NET/Storage/MetaDataStore/DataRepository.cs

```diff
@@ -46,6 +52,22 @@ public async Task LogDataAsync(Guid runId, IDataView dataView)
             }
         }
 
+        private string GetStringHashFromDataView(IDataView dataView)
+        {
+            string hash = "";
+            using (SHA256 sha256Hash = SHA256.Create())
+            {
+                using (var stream = new MemoryStream())
+                {
+                    new MLContext().Data.SaveAsBinary(dataView, stream);
+                    var bytes = sha256Hash.ComputeHash(stream);
+                    hash = Convert.ToBase64String(bytes);
```

Comment:
No need to initialize a separate variable here, we can just do `return Convert.ToBase64String(bytes)`